exercise/exercise_models.py

```diff
@@ -533,6 +533,11 @@ class BaseExercise(LearningObject):
         default=GRADING_MODE.BEST,
         help_text=_('BASE_EXERCISE_GRADING_MODE_HELPTEXT'),
     )
+    regrade_task = models.CharField(
```

Comment:
One option would be to define the task identifier in a new model so that it doesn't pollute the existing exercises. If the field is not really an attribute of (~all) exercises, then why should it be part of the same model?

If there were a new model class, it would refer to the exercise with a ForeignKey.